src/components/Header/Header.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useHistory } from "react-router";
+
+import Navigation from "../Navigation";
+
+import { logout } from "../../store/actions/authActions/logout";
+import { authStateSelector } from "../../store/reducers/authReducer/selectors";
+
+import { NavLinks } from "../../types/navLinks";
+
+const Header: React.FC = (): JSX.Element => {
+  const dispatch = useDispatch();
+  const { token } = useSelector(authStateSelector);
+  const history = useHistory();
+
+  const logOut = () => {
+    dispatch(logout());
```

Comment:
If your request has failed what we see ? `  history.push(NavLinks.default);` will be work ?